test/functional/api_keys_controller_test.rb

```diff
@@ -223,7 +223,7 @@ class ApiKeysControllerTest < ActionController::TestCase
         end
 
         should "show error to user" do
-          page.assert_text "Show dashboard scope must be enabled exclusively"
+          assert page.has_content? "Show dashboard scope must be enabled exclusively"
```

Comment:
these changes actively make assertion failure worse, can we migrate at least to the capybara minitest methods to maintain the good error messages?

Reply:
That's the case I have explained in the description. It is not possible to simply including capybara assertions since they do conflict with rails-dom-testing gem. For example `assert_select` does different kind of assertion. Happy to move those tests to be integration ones in next PR and remove rails-dom-testing dependency (maybe rails-controller-testing in general) at all.

Reply:
Alternativelly I can manually port assert_content assertion to functional tests to cover most of the similar changes to report with nice and friendly message before mentioned refactoring.

Reply:
let's manually port the one or two methods we need?

Reply:
OK, I'll do.

Reply:
@segiddins I did my best to port at least something to keep functional assertions friendly.